doc/topics/sdb/index.rst

```diff
@@ -79,6 +79,20 @@ from the ``kevinopenstack`` profile above, you would use:
 
     salt-call sdb.get sdb://kevinopenstack/password
 
+To get SDB sub-keys, ie
+
+.. code-block:: yaml
+
+    users:
+      user1:
+        id: 12345
+
+use a colon to separate sub key values
+
+.. code-block:: bash
+
+    salt-call sdb.get sdb://users:user1:id
+
```

Comment:
@perfecto25 , I use the conventional comments method of giving feedback on pull requests: https://conventionalcomments.org/

**Praise:** Thanks for your docs pull request! It seems like there's definitely some missing documentation about how to get SDK sub-keys, so I'm really grateful you're adding it.

**Question (blocking, clarification):** This might be a product of my own misunderstanding of how SDB keys work, but I'm confused about the relationship between the state file (SLS) example and the CLI example. Should that format you give in the CLI example also be used in the SLS example?

**Suggestion (blocking, technical writing best practices):** In technical documentation, it's a little unusual to break up a sentence to insert an example and then finish the sentence after the the example. I'd recommend changing the flow of information to avoid that awkward mid-sentence break. How about this instead:

```
To get SDB sub-keys from the CLI, use a colon to separate sub key values. For example: 

.. code-block:: bash
    salt-call sdb.get sdb://users:user1:id

To get SDB sub-keys in a state file, use this syntax:

.. code-block:: yaml
    users:
      user1:
        id: sdb.get sdb://users:user1:id
        
```

Obviously that last example :point_up: is me trying to wrap my head around about the relationship between the CLI and SLS examples. Feel free to correct it if I've gotten it wrong.

Reply:
@perfecto25 , thanks for the explanation! Could you also make that change I recommended just modifying the sentence so that it's not broken up across the example?